File: site/static/templates/blank_play_state/java/teavm/src/main/java/{{PACKAGE_PATH}}/teavm/{{GAME}}TeaVMLauncher.java
package {{PACKAGE}}.teavm;

import org.flixelgdx.backend.teavm.FlixelTeaVMLauncher;
import {{PACKAGE}}.{{GAME}}Game;

/**
 * Browser entry point.
 * <p>
 * TeaVM turns your JVM bytecode into JavaScript.
 * <p>
 * FlixelTeaVMLauncher connects that web runtime to your FlixelGame.
 */
public final class {{GAME}}TeaVMLauncher {

  public static void main(String[] args) {
    FlixelTeaVMLauncher.launch(new {{GAME}}Game());
  }

  private {{GAME}}TeaVMLauncher() {}
}

File: site/static/templates/blank_play_state/kotlin/teavm/src/main/kotlin/{{PACKAGE_PATH}}/teavm/{{GAME}}TeaVMLauncher.kt
package {{PACKAGE}}.teavm

import org.flixelgdx.backend.teavm.FlixelTeaVMLauncher
import {{PACKAGE}}.{{GAME}}Game

/**
 * Browser entry point.
 *
 * TeaVM turns your JVM bytecode into JavaScript.
 *
 * FlixelTeaVMLauncher connects that web runtime to your FlixelGame.
 */
fun main() {
  FlixelTeaVMLauncher.launch({{GAME}}Game())
}
